client/src/partials/users/userPermission.modal.html

```diff
@@ -0,0 +1,41 @@
+<form name="PermissionForm" bh-submit="UserPermissionModalCtrl.submit()">
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="title">
+        {{ "USERS.PERMISSIONS" | translate }}
+        <label class="label label-warning">{{ "USERS.PERMISSIONS" | translate }}</label>
+        <label class="label label-warning">{{ UserPermissionModalCtrl.user.display_name }} ({{ UserPermissionModalCtrl.user.username }})</label>
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body">
+    <ul class="list-group" style="max-height: 600px; overflow:auto;">
+      <li class="list-group-item">
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" ng-model="UserPermissionModalCtrl.super" ng-change="UserPermissionModalCtrl.toggleSuperUserPermissions(UserPermissionModalCtrl.super)">
+            {{ "USERS.SUPER" | translate }}
```

Comment:
This would look much better if it were **bold**.  That way it looks important - at the moment, it just looks like any of of the top level folders.